Make test failure under macOS

<!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->


**What happened**:
```bash
# make test
find "./pkg/" -name "*_test.go"|xargs -i dirname {}|uniq|xargs -i go test  {}
xargs: illegal option -- i
xargs: illegal option -- i
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
make: *** [test] Error 1
```
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeEdge version:
- Hardware configuration:
- OS (e.g. from /etc/os-release): macOS
- Kernel (e.g. `uname -a`): Darwin
- Others:

--- a/edge/Makefile
+++ b/edge/Makefile
@@ -14,7 +14,7 @@ endif
 
 export GOARCHAIUS_CONFIG_PATH=$(CURDIR)
 test:
-	find ${TEST_DIR} -name "*_test.go"|xargs -i dirname {}|uniq|xargs -i go test ${T} {}
+	find ${TEST_DIR} -name "*_test.go"| xargs -I{} dirname {} | uniq | xargs -I{} go test ${T} {}
 
 # verify
 .PHONY: verify

--- a/edge/pkg/edged/containers/container_manager.go
+++ b/edge/pkg/edged/containers/container_manager.go
@@ -1,5 +1,3 @@
-// +build linux
-
 /*
 Copyright 2015 The Kubernetes Authors.
 

